server/rest/annotation.py

```diff
@@ -141,16 +144,63 @@ def getAnnotationSchema(self, params):
     )
     @access.cookie
     @access.public
-    @filtermodel(model='annotation', plugin='large_image')
+    # @filtermodel(model='annotation', plugin='large_image')
```

Comment:
Is this needed?

Reply:
No.  I could remove the line or comment it further.  `@filtermodel` doesn't work with a function that returns a generator rather than returning a complete result.  The filtering is now done within the function.

Reply:
Ah, I see.  I think removing it would be better.